docs/api/hooks.md

```diff
@@ -404,17 +402,14 @@ import { bindActionCreators } from 'redux'
 import { useDispatch } from 'react-redux'
 import { useMemo } from 'react'
 
-export function useActions(actions, deps) {
+export function useActions(actions, deps = []) {
   const dispatch = useDispatch()
-  return useMemo(
-    () => {
-      if (Array.isArray(actions)) {
-        return actions.map((a) => bindActionCreators(a, dispatch))
-      }
-      return bindActionCreators(actions, dispatch)
-    },
-    deps ? [dispatch, ...deps] : [dispatch]
-  )
+  return useMemo(() => {
+    if (Array.isArray(actions)) {
+      return actions.map((a) => bindActionCreators(a, dispatch))
+    }
+    return bindActionCreators(actions, dispatch)
+  }, [dispatch, ...deps])
```

Comment:
This is also a micro-optimization removal. Spread operator performs operations every time (even when deps wasn't given).
In some projects, deps are not given to useActions because most of the time they are not needed.

However, we can reduce the additional cost of this refactor (in V8) by replacing the elements in this new array:
```
[dispatch, ...deps] // bad
[...deps, dispatch] // good
```
If you don't believe it, fire up chrome profiler ;)